### PR TITLE
Add new study: Britton_1994

### PR DESCRIPTION
**Pull request by ehwenk**
* add new study, transcribed from literature
* only trait is `recruitment_time`, which doesn't generate in reports (because categorical, but without specified values), so report includes only partial information

### CODE REVIEW

File: data/Britton_1994/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    to allow movement of water from the tanks into the trays. Artificial lighting
+    or heating was not used in the glasshouse, although a cooling system, which came
+    into operation at around 27OC, maintained temperatures in the glasshouse below
+    30°C for most of the year. At the end of the eight-week wetting period, individuals
+    that had germinated were identified and counted. The sediments were then discarded.
+    Maximum-minimum thermometers were used to measure air temperatures and water temperatures
+    within the tanks, both in the glasshouse and outside. Readings were taken at approximately
+    three-day intervals throughout each germination trial. Light measurements were
+    recorded at midday at the same intervals with the aid of a Li-Cor (LI-185B) photometer,
+    which measures radiation in the 400 to 700 nm waveband. Data obtained in the first
```
Comment (yangsophieee) [inline data/Britton_1994/metadata.yml:128]: Maybe remove the rest of the methods from this sentence (and for the trait below)?

File: data/Britton_1994/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    species at each site in the two spring trials (spring 1 and spring 2) were carried
+    out with Student's t-test.
+- var_in: germination_outdoors
+  unit_in: .na
+  trait_name: recruitment_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: 30
+  basis_of_record: field_experiment
```
Comment (yangsophieee) [inline data/Britton_1994/metadata.yml:142]: Is this technically `captive_cultivated` if it was on the grounds of the university? Also does that mean that the site should be the University of New England and there should be a context variable for seed provenance (wetlands in northern New South Wales)?

**Review by yangsophieee — APPROVED** [review]
First study of recruitment_time in AusTraits, nice find! 
I've fixed two small transcription errors in the data.
Also I think there should be a test that checks if `recruitment_time`, `flowering_time`, etc. have 12 characters in the string as I only thought to manually check it and found that there were 13 characters (there was a mistake in custom_R_code which I fixed). (Should I raise this in the Issues tab?)

File: data/Britton_1994/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/Britton_1994/metadata.yml:128]: Done

File: data/Britton_1994/metadata.yml (hunk 2 above)
Comment (ehwenk) [reply data/Britton_1994/metadata.yml]: Captive cultivated is meant for an arboretum or common garden experiment - I think for seed germination it really is just an experiment. But I did switch the seed provenance to a context property and re-did location.

**Pull request merged**